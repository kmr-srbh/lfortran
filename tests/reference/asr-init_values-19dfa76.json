{
    "basename": "asr-init_values-19dfa76",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/init_values.f90",
    "infile_hash": "1cafced706fa2754f92f2d460867d6c84638fd10447df12f124fd419",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-init_values-19dfa76.stdout",
    "stdout_hash": "3cf2343fbf64d9f24aec91c1edc08b2cde425a09ef4a77dbc6508ce4",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}