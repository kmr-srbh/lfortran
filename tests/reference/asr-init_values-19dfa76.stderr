semantic error: The RHS character len must be known at compile time
  --> tests/../integration_tests/init_values.f90:10:5
   |
10 |     character(len = 1), parameter :: s1 = "l"
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ 
